wallet/test/de/schildbach/wallet/util/viewModels/MainViewModelTest.kt

```diff
@@ -200,16 +216,35 @@ class MainViewModelTest {
         every { savedStateMock.set<TxFilterType>(any(), any()) } just runs
     }
 
-    @Test
+    @Test(timeout = 1000)
     fun observeBlockchainState_replaying_notSynced() {
         every { blockchainStateMock.observeState() } returns MutableStateFlow(BlockchainState(replaying = true))
 
         val viewModel = spyk(
             MainViewModel(
-                analyticsService, configMock, uiConfigMock,
-                exchangeRatesMock, walletDataMock, walletApp, platformRepo,
-                mockk(), mockk(), blockchainIdentityConfigMock, savedStateMock, transactionMetadataMock,
-                blockchainStateMock, mockk(), mockk(), mockk(), userAgentDaoMock, mockk(), mockDashPayConfig, mockk(), mockk()
+                analyticsService,
+                configMock,
+                uiConfigMock,
+                exchangeRatesMock,
+                walletDataMock,
+                walletApp,
+                platformRepo,
+                platformService,
+                platformSyncService,
+                blockchainIdentityConfigMock,
+                savedStateMock,
+                transactionMetadataMock,
+                blockchainStateMock,
+                biometricHelper,
+                telephonyManager,
+                invitationsDaoMock,
+                userAgentDaoMock,
+                dashPayProfileDaoMock,
+                mockDashPayConfig,
+                dashPayContactRequestDao,
+                coinJoinConfig,
+                coinJoinService,
+                crowdNodeApi
```

Comment:
All arguments have names to avoid ambiguity with `mockk()`

Nevertheless, the tests still do not complete -- there must be another class that has problems.